resources/prosody-plugins/mod_muc_moderator_hidden.lua

```diff
@@ -0,0 +1,36 @@
+--This module adds is_hidden tag when the user from jwt is moderator and when
+--jwt contains `hidden` flag set to true, or removes it in case it was
+--added maliciously from client sent presence.
+--The module must be enabled under the muc component.
+local is_user_hidden = module:require "util".is_user_hidden;
+local tag_name = "is_hidden"

```

Comment:
Isn't this the same as https://github.com/jitsi/jitsi-meet/blob/06bd279f225e2a6ea0c7121ca184e1c124a0d5f9/resources/prosody-plugins/mod_muc_poltergeist.lua#L146
Which we already handle in lib-jitsi-meet code?

Reply:
Not sure. The target here is for a user to join but for the other users not see this user. This is for the "secretary manages the slides" scenario. Does poltergeist accomplish the same?

Reply:
Maybe, I don't know what does this flag do. In the other case we ignore presence so we do not see a participant leave and join. I suppose both end up doing the same.

Reply:
Ah you need the presence actually to be able to do certain operations.